### PR TITLE
Built airflow dag for weather  dashboard

### PR DESCRIPTION
**Pull request by vsvsharma**

### CODE REVIEW

File: weather_dashboard_airflow.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+    kwargs['ti'].xcom_push(key="weather_data", value=weather_data_list)
+
+def process_location_data(ti):
+    """
+    In this function, location data is extracted from the whole weather data that was
+    pushed into the xcom in the exraction data process.
+    in this city iso codes are declared to so that 
+    cities can be plotted on a map in the dashboard that will be built later on.
+
```
Comment (nvnvashisth) [inline weather_dashboard_airflow.py:68]: remove this space

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
-# weather_dashboard_airflow
```
Comment (nvnvashisth) [inline README.md:1]: remove this line

File: weather_dashboard_airflow.py [hunk 3]
```diff
@@ -0,0 +1,261 @@
+#Import necessary libraries
```
Comment (nvnvashisth) [inline weather_dashboard_airflow.py:1]: give space between # and import

**Review by nvnvashisth — COMMENTED** [review]
fix it please

File: README.md (hunk 2 above)
Comment (vsvsharma) [reply README.md:1]: fixed

File: weather_dashboard_airflow.py (hunk 3 above)
Comment (vsvsharma) [reply weather_dashboard_airflow.py:1]: fixed

File: weather_dashboard_airflow.py (hunk 1 above)
Comment (vsvsharma) [reply weather_dashboard_airflow.py:68]: fied

**Review by vsvsharma — COMMENTED** [review]
all changes are done

**Review by nvnvashisth — APPROVED** [review]
Looks good

**Pull request merged**